jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -80,6 +80,13 @@
 	// Instance variables
 	private long threadRef;									// Used by the VM
 	long stackSize = 0;
+    /* For Java 14+:
+     * deadInterrupt tracks the thread interrupt state when threadRef has no reference (ie thread is not alive). 
+     * Note that this value need not be updated while the thread is running since the interrupt state will be 
+     * tracked by the vm during that time. Because of this the value should not be used over calling
+     *  isInterrupted() or interrupted().
+     */
+	private volatile boolean deadInterrupt;
```

Comment:
Why bother introducing this variable at all in versions < 14? In the vmconstantpool.xml the versions can be specified via `versions="14-"`. Similarly the VM code can use `#if JAVA_SPEC_VERSION >= 14`